syz-ci/syz-ci.go

```diff
@@ -265,6 +271,34 @@ func main() {
 	}
 }
 
+func deprecateAssets(cfg *Config, shutdownPending chan struct{}) {
+	dash, err := dashapi.New(cfg.DashboardClient, cfg.DashboardAddr, cfg.DashboardKey)
+	if err != nil {
+		log.Logf(0, "deprecateAssets: failed to create dashapi: %v", err)
+		return
+	}
+	storage, err := asset.StorageFromConfig(cfg.AssetStorage, dash)
+	if err != nil {
+		log.Logf(0, "deprecateAssets: failed to create asset storage: %v", err)
+		return
+	}
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
If we need to do this, this needs to be done in main. Otherwise different small functions everywhere will re-seed it and log different seeds, which will be confusing. This is a global things, so it needs to be setup in main.
Or better use a local rand instance.
But also why do we need this randomization here? Why not run it, say, once per day?

Reply:
The reasoning was to avoid (almost) synchronous requests from a lot of syz-ci's to the dashboard.

Though now that deprecated assets querying was turned into the much faster needed assets querying, it's probably no longer so important indeed. I'll remove the randomization.